modules/ducktests/tests/ignitetest/services/utils/ignite_aware.py

```diff
@@ -47,6 +47,10 @@ def __init__(self, context, config, num_nodes, **kwargs):
 
         self.spec = resolve_spec(self, context, config, **kwargs)
 
+        # Force eager SSH client creation.
+        for node in self.nodes:
+            node.account.ssh("/bin/true")
```

Comment:
Can be replaced with `node.account.ssh_client`, it is enough to client will be initialized

Reply:
Done.